models/user.go

```diff
@@ -119,6 +121,11 @@ func (u *User) BeforeUpdate() {
 	u.UpdatedUnix = time.Now().Unix()
 }
 
+// Set time to last login
+func (u *User) UpdateLastLogin() {
```

Comment:
Maybe name this `SetLastLogin()`? `Update` is a bit misleading, it would be apropriate if this actually saved the change to the database.